app/components/post_draft/quick_actions/file_quick_action/index.tsx

```diff
@@ -83,7 +83,7 @@ const FileQuickAction = ({disabled, fileCount = 0, intl, maxFileCount, onUploadF
         >
             <CompassIcon
                 color={color}
-                name='file-document-outline'
+                name='file-text-outline'
```

Comment:
@matthewbirtch it seems the `file-document-outline` is no longer included in compassIcon, @jespino made this change and for this PR is fine.. but I want to make sure this:
1. works fine for now
2. We submit a PR to use the right icon for this

Reply:
yes, sorry, the name changed to `file-text-outline`

Reply:
Great, looks like the change is right :)